### DISCUSSION TITLE
Kral dük kimdir sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#298] [Zeka Sorusu] [answered]
Bir kral, bir dük, bir kont ve bir lord dörtköşe masa etrafında briç oynuyorlar. İsimleri Louis, Jan, Serge ve Albert’dir. Louis kraldır ve Jan’ın solundadır. Serge lordun sağındadır. Jan’ın karşısında oturan Albert, kont değildir. Serge nedir?

**Comment by Mertcakm** [comment]
Kont

**Reply by sonsuzus** [reply]
Soruları açıklamalı çözelim.

**Comment by adasue** [comment, marked as answer]
Louis'in kral olduğunu biliyoruz masada rastgele bir yere koyuyoruz soluna da Jan'ı koyuyoruz. Serge lordun sağındaymış ama lordun kim olduğunu şu anlık bilmiyoruz. Jan'ın karşısında da Albert oturuyormuş. O zaman Jan'ın solu ve önü dolu olduğu için sağına da Serge'nin geleceğini çıkarabiliyoruz. Serge de lordun sağında olduğu için Jan'ın lord olduğunu buradan çıkarıyoruz. Albert da kont olamayacağı için ona dük diyoruz. En sona da kont kalıyor o da Serge. Yani sıralama şu şekilde:
Louis->kral
Jan->lord
Albert->dük
**Serge->kont**
